docs/knowledge_base/threat_feed.md

```diff
@@ -97,3 +97,71 @@ For example, if you want to limit the items per page to 3 since July 19, 2023 yo
 ```text
 https://threats.phylum.io/?per_page=3&since=2023-07-19
 ```
+
+## Package Lookup Endpoint
+
+The threat feed API provides a package lookup endpoint that details information about a specific package, including its malware status, risk scores, and a list of identified issues. This endpoint is designed to fetch information based on a combination of ecosystem, package name, and version.
+
+### Endpoint
+
+`GET https://threats.phylum.io/<ecosystem>/<name>/<version>`
+
+### Parameters
+
+- `ecosystem`: The package ecosystem (one of: pypi, npm, maven, rubygems, nuget, golang, cargo).
+- `name`: The name of the package.
+- `version`: The version of the package.
+
+### Response Structure
+
+The response is a JSON object containing the following keys:
+
+- `code`: The HTTP status code.
+- `malware`: A boolean indicating if the package is has been triaged as malware.
+- `riskScores`: An object containing risk scores for each of the 5 Phylum risk domains associated with the package.
+- `issues`: An object listing all identified issues and vulnerabilities related to the package. The details included with each issue will depend on the nature of the issue.
+
+#### Issues Object
+
+Each entry in the `issues` object is keyed by the issue type (e.g., vulnerability identifier, rule name) and contains an object with the following details:
+
+For vulnerabilities:
+
+- `title`: The title of the vulnerability.
+- `overview`: A brief overview of the vulnerability.
+- `severity`: The severity level of the vulnerability.
+- `recommendation`: Recommendations for addressing the vulnerability.
+
+For rules, the details included will depend on the type of rule. For example, the `Compiled Binaries Rule` will include a list of the binary files and their paths within the package. The `Cargo Build File Rule` will simply contain a Boolean value of `true` indicating that a `build.rs` file is present in the package.
+
+### Usage Example
+
+Request:
+
+`curl https://threats.phylum.io/pypi/adrmdr/0.1.0 -H "Authorization: Bearer $PHYLUM_API"`
```

Comment:
Suggest changing this to a fenced code block (i.e., ```bash) to match the style from a similar command earlier in this document so it looks like this:

```bash
curl https://threats.phylum.io/pypi/adrmdr/0.1.0 -H "Authorization: Bearer $PHYLUM_API"
```

instead of this:

`curl https://threats.phylum.io/pypi/adrmdr/0.1.0 -H "Authorization: Bearer $PHYLUM_API"`

Reply:
Will fix.